.env.example

```diff
@@ -1,3 +1,4 @@
+USE_SYSTEM_ENV=true
```

Comment:
@fereira Are you planning on retaining this switch (fallback to `application.properties`) moving forward?

Reply:
I don't have to, but if I end up migrating from jetty to spring-boot that's the more standard (default) way of reading in properties.  For local development I prefer not to have to deploy as a container, and just to start the app using maven.